modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -133,11 +133,12 @@ export default class ColorInput extends React.Component<ColorInputProps> {
           inputRef={inputRef}
           onChange={this.handleChange}
           type="text"
-          placeholder={placeholder}
+          placeholder={value ? '' : placeholder}
           value={formattedValue}
           error={error}
           spellCheck={false}
           disabled={disabled}
+          // aria-label={!value ? 'Label edit blank hex value' : `edit hex value ${formattedValue}`}
```

Comment:
For more clarity, can I get some thoughts on this potential fix? It indicated that no only is it and editable text but the text should be a hex value? Thoughts?

Reply:
With this enabled it announces when no value:
`Label edit blank hex value, edit text FFFFFF`

Reply:
With value  is e6e it announces:
`e6e insertion at end of text, edit hex value e6e, edit text`

Reply:
Nvm, i realized this might conflict with the `for` attribute